src/runtime_src/core/tools/common/ReportHost.cpp

```diff
@@ -70,7 +70,7 @@ ReportHost::writeReport(const xrt_core::device * _pDevice,
     _output << boost::format("  %-20s : %s\n") % "Release" % _pt.get<std::string>("host.os.release");
     _output << boost::format("  %-20s : %s\n") % "Version" % _pt.get<std::string>("host.os.version");
     _output << boost::format("  %-20s : %s\n") % "Machine" % _pt.get<std::string>("host.os.machine");
-    _output << boost::format("  %-20s : %s\n") % "Distribution" % _pt.get<std::string>("host.os.distribution");
+    _output << boost::format("  %-20s : %s\n") % "Distribution" % _pt.get<std::string>("host.os.distribution","N/A");

```

Comment:
Being that the "Distribution" value doesn't make sense here, should we instead simply not produce the output.

Reply:
its an common code for pcie and edge. i dont have an way to show in one and not other.